app/models/file_attachment/revision.rb

```diff
@@ -19,6 +19,7 @@ class FileAttachment::Revision < ApplicationRecord
            :asset,
            :assets,
            :ensure_assets,
+           :blob,
```

Comment:
does this need to be content_type and byte_size?

Reply:
Sorry - fixed.